frontend/src/core/placeable/parsers/pythonFormattingVariable.test.js

```diff
@@ -15,15 +15,28 @@ describe('pythonFormattingVariable', () => {
         ['%(cows.number)d', 'There were %(cows.number)d cows'],
         ['%(number of cows)d', 'There were %(number of cows)d cows'],
         ['%(number)03d', 'There were %(number)03d cows'],
-        ['%(number) 3d', 'There were %(number) 3d cows'],
         ['%(number)*d', 'There were %(number)*d cows'],
         ['%(number)3.1d', 'There were %(number)3.1d cows'],
         ['%(number)Ld', 'There were %(number)Ld cows'],
+        ['%s', 'path/to/file_%s.png'],
+        ['%s', 'path/to/%sfile.png'],
     ])
     .it('marks `%s` in `%s`', (mark, content) => {
         const Marker = createMarker([pythonFormattingVariable]);
         const wrapper = shallow(<Marker>{ content }</Marker>);
         expect(wrapper.find('mark')).toHaveLength(1);
         expect(wrapper.find('mark').text()).toEqual(mark);
     });
+
+    each([
+        ['10 % complete'],
+        // We used to match '%(number) 3d' here, but don't anymore to avoid
+        // false positives.
```

Comment:
Nit: A ref to the bug would be helpful.

Reply:
BTW: sadly the issue reported in https://bugzilla.mozilla.org/show_bug.cgi?id=1251186#c0 is not fixed by this changeset, because the same placeable gets falsely detected by the `stringFormattingVariable`. (I also check that no other parser detects it.)

Reply:
Woopsy, thanks for catching that!